src/worker/abstract-worker.ts

```diff
@@ -22,26 +23,21 @@ export abstract class AbstractWorker<
   Response = unknown
 > extends AsyncResource {
   /**
-   * The maximum time to keep this worker alive while idle. The pool automatically checks and terminates this worker when the time expires.
+   * Id of the last task processed by this worker.
    */
-  protected readonly maxInactiveTime: number
+  protected lastTaskId: number
   /**
-   * The kill behavior set as option on the Worker constructor or a default value.
+   * Timestamp of the last task processed by this worker.
    */
-  protected readonly killBehavior: KillBehavior
+  protected lastTaskTimestamp: number
   /**
-   * Whether the worker is working asynchronously or not.
+   * Handler ID of the `aliveInterval` alive check.
    */
-  protected readonly async: boolean
+  protected readonly aliveInterval?: NodeJS.Timeout
```

Comment:
Already there before. It's a native NodeJS type put by you.

Reply:
![image](https://user-images.githubusercontent.com/7195563/118364913-d189eb80-b59a-11eb-9834-5a0555e576ff.png)

What do you talking about?!
You switched it from `boolean` to `NodeJS.Timeout`
I just asked if there is a better solution than using this global declarations
IMO global declarations coming out of nowhere is kinda a bad design, cause suddenly it could break with a version change of NodeJS

We could use `ReturnType<typeof setTimeout>` as described in https://stackoverflow.com/questions/45802988/typescript-use-correct-version-of-settimeout-node-vs-window, but yes I see that would be overkill
So seems we need to live with this based on NodeJS's decision :confused:

Reply:
Reread the diff, variable is not the same ;) 
And native NodeJS types are unlikely to change, especially for `setInterval()` that was here years before node.js was created.

Reply:
Nevermind :facepalm: 
The diff really confused me, cause there were so many changes